environments/gym_wrapper.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 AI Redefined Inc. <dev+cogment@ai-r.com>
```

Comment:
Could you explain why we need this or the differences with gym_adapter?

Reply:
It is because in the `gym_adapter.py` leans towards the dAgger algorithm where it has TEACHER actor et RL actor. @cloderic asked me to implement a vanilla version for gym envs (I basically remove TEACHER  actor).